src/test/java/pl/psnc/dei/util/RecordIdValidatorTest.java

```diff
@@ -0,0 +1,38 @@
+package pl.psnc.dei.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+public class RecordIdValidatorTest {
+
+	@Test
+	public void shouldValidateRecordId() {
+		String testRecordId1 = "/12345/12345_my_record";
+		String testRecordId2 = "/12345678/12345";
+		String testRecordId3 = "/12345/_my_record_";
+		String testRecordId4 = "/123/record";
+
+		Assert.assertTrue(RecordIdValidator.validate(testRecordId1));
+		Assert.assertTrue(RecordIdValidator.validate(testRecordId2));
+		Assert.assertTrue(RecordIdValidator.validate(testRecordId3));
+		Assert.assertTrue(RecordIdValidator.validate(testRecordId4));
+	}
+
+	@Test
+	public void shouldNotValidateRecordId() {
+		String testRecordId1 = "/12345_12345_my_record";
+		String testRecordId2 = "/12345/my-record";
```

Comment:
Why is this identifier invalid? I do not see any limitation in case of "-" sign in LOCAL_ID part of Europeana IRI (https://pro.europeana.eu/resources/apis/intro#identifying-records)

Reply:
Pattern updated

Reply:
Pattern uses "visible characters" now, so it should be ok. Approved!